### PR TITLE
Change the recommended storage statement

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
This is to be cherry-picked to enterprise-4.1 and enterprise-4.2.

File: monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc [hunk 1]
```diff
@@ -29,7 +29,7 @@ Running cluster monitoring with persistent storage means that your metrics are s
 
 [IMPORTANT]
 ====
-In {product-title} 4.1 deployed on bare metal, Prometheus and Alertmanager cannot have persistent storage and are ephemeral only. For use cases other than bare metal, you must use dynamic provisioning to use persistent storage. In particular, do not use NFS for the monitoring stack.
+See xref:../../scalability_and_performance/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage[Recommended configurable storage technology].
```
Comment (ahardin-rh) [inline monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc]: s/.html/.adoc

Comment (rh-max) [reply monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc]: Good catch, implemented.

Comment (ahardin-rh) [reply monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc]: Thanks! We'll just need the commits squashed into one, then I can merge and cherry-pick.

Comment (rh-max) [reply monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc]: Squashed. Thank you.

**Pull request merged**